app/assets/javascripts/discourse/app/lib/uploads.js

```diff
@@ -283,27 +281,14 @@ export function getUploadMarkdown(upload) {
 
 export function displayErrorForUpload(data, siteSettings) {
```

Comment:
Likewise this will be able to be deleted in the future once we aren't using JQFU